ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'FRC=$?; createUpgradeJunit; debug' EXIT TERM
```

Comment:
this should be `createArchMigrationJunit`

Reply:
ah, right. Thank you Ales!